Adafruit_IO_Power_Relay/code.py

```diff
@@ -107,8 +106,8 @@ def on_relay_msg(client, topic, value):
 # Set up a MiniMQTT Client
 client = MQTT.MQTT(
     broker="io.adafruit.com",
-    username=secrets["aio_username"],
-    password=secrets["aio_key"],
+    username=os.getenv("AIO_USERNAME"),
```

Comment:
@dhalbert `aio_username` is changed to to `AIO_USERNAME` (uppercase here) as well as `aio_key`/`AIO_KEY`. 

Is this lcase to ucase intentional for the TOML files? Is it referenced by the uppercase variable name in the guide?

Reply:
Uppercase is what we have been using as a standard. I changed the guides to use uppercase names.

There has been some discussion of `ADAFRUIT_AIO_*`, but `AIO_*` seems fine and is used in most guides.

Reply:
> I changed the guides to use uppercase names.

Sounds good1